index.html

```diff
@@ -1,17 +1,26 @@
 <!DOCTYPE html>
 <html>
 <head>
+<script src="next_paint_hack.js"></script>
 <script src="performance_timeline_emit.js"></script>
 <script src="event_timing.js"></script>
 <script>
-
-const po = new PerformanceObserver((e) => {
-  for (let entry of e.getEntries()) {
-    console.log(entry);
-  }
-});
-po.observe({entryTypes:["event"]})
-
+  let cntEntry = 0;
+  const po = new PerformanceObserver((e) => {
+    for (let entry of e.getEntries()) {
+      cntEntry++;
+      if (entry.nextPaint) {
+        const paintsInRange = paintTimes.filter(p => p > entry.processingStart && p < entry.nextPaint);
```

Comment:
I'm not clear on what this is doing.

Reply:
It's based on my earlier assumption that between processing end and FP should there be only one rAF, but the assumption was wrong because rAF can happen while mouse move is queued.